src/AST-Core/RBLiteralNode.class.st

```diff
@@ -28,6 +28,11 @@ RBLiteralNode >> = anObject [
 	^self class = anObject class
 ]
 
+{ #category : #visiting }
+RBLiteralNode >> acceptVisitor: aProgramNodeVisitor [
+	^aProgramNodeVisitor visitLiteralNode:self
```

Comment:
Please use the code formatter
^ aProgramNodeVisitor visitLiteralNode: self

Reply:
Done